### PR TITLE
docs: extended contributing requirements

### PR DESCRIPTION
**Pull request by kirrg001**
Discovered in #1358

### CODE REVIEW

**Comment by kirrg001** [comment]
@aryamohanan @abhilash-sivan Please execute locally & approve.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ Python3 (< 3.11) is required, otherwise the db2 package won't build, see https:/
 If you're having issues with distutils module, then double check the python3 version (python3 --version) and make sure that the version pointing to is < 3.12
 The error can be something like: ModuleNotFoundError: No module named 'distutils'
 
+`npm i lerna@6.1.4 -g`
```
[the remark this answers is not in the document]
Comment (aryamohanan) [reply CONTRIBUTING.md]: Sure

**Comment by kirrg001** [comment]
See https://github.com/instana/nodejs/pull/1358/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R43

**Review by abhilash-sivan — APPROVED** [review]

**Comment by aryamohanan** [comment]
I could see a contradictory statement here https://github.com/instana/nodejs/pull/1378/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R127

**Comment by kirrg001** [comment]
> I could see a contradictory statement here https://github.com/instana/nodejs/pull/1378/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R127

Good catch 👍 
Fixed https://github.com/instana/nodejs/commit/dcc4b744ad1a4a538bf20d05b20fedf301d99e05

**Pull request merged**